pkg/virtctl/vmexport/vmexport.go

```diff
@@ -70,6 +71,8 @@ const (
 	SNAPSHOT_FLAG       = "--snapshot"
 	INSECURE_FLAG       = "--insecure"
 	KEEP_FLAG           = "--keep-vme"
+	RAW_FLAG            = "--raw"
+	DECOMPRESS_FLAG     = "--decompress"
```

Comment:
I'm a little confused here.  Why two options?  Don't they ultimately bean the same thing?

What do you think of this, forget about these to flags for a sec.  What if we:

1) always fetch the gzip image
2) ad "-o" and/or "--output" arg that accepts "gzip" or "raw" (gzip is default to be consistent with current invocation)
3) if raw is specified, unzip the image while downloading

Reply:
So I initially wanted to allow both downloading the raw file (--raw) and downloading and decompressing the gzipped one (--decompress). I know they produce the same output but thought it was nice to allow both ways (some users might care about downloading bigger files, others won't). I also prefer boolean flags since they are more immediate and less confusing. But if you prefer to have a single flag to specify the output I can change it, just wanted to share the rationale behind this design.

Reply:
I think this is a case of too many knobs leading to user confusion.

Reply:
Okay so I added a `--format` flag to specify either raw or gzip. `--output` is already a flag so couldn't use that one.